climada_petals/entity/exposures/openstreetmap/osmconf.ini

```diff
@@ -0,0 +1,121 @@
+#
+# Configuration file for OSM import
+#
+
+# put here the name of keys, or key=value, for ways that are assumed to be polygons if they are closed
+# see http://wiki.openstreetmap.org/wiki/Map_Features
+closed_ways_are_polygons=aeroway=aerodrome,amenity,boundary,building,craft,geological,historic,healthcare=hospital,healthcare=clinic,healthcare=doctors,healthcare=dentist,landuse,leisure,military,natural,office,place,shop,sport,tourism,highway=platform,public_transport=platform,reservoir_type,tower_type,power=generator,power=substation,power=transformer,power=terminal,power=plant,power=tower,man_made=wastewater_plant,man_made=wastewater_tank,utility,water,building=school,amenity=school,building=kindergarten,amenity=kindergarten,building=college,amenity=college,building=university,amenity=university,building=college,amenity=college,building=childcare,amenity=childcare,amenity=hospital,amenity=clinic,amenity=doctors,amenity=pharmacy,amenty=nursing_home,amenity=dentist,telecom=central_office,telecom=data_center,telecom=street_cabinet,railway=platform,railway=station,man_made=storage_tank,man_made=oil_refinery,man_made=petroleum_well,shop=supermarket,shop=grocery,shop=greengrocer,shop=general,shop=bakery,pipeline=substation,public_transport
```

Comment:
Could we have more context? Did you build this list from that URL? Is it complete (as of 2021), or might I need to add to it?

In general, I'd like have a sense of
(1) when will I have to edit this file (e.g. does this just work for critical infrastructure/common features, or is it complete enough only to need updating when OSM standards change?)
(2) if so, what will i need to change to add/update a feature or customise for my purposes?

If multiple users might be editing this and it would need more than a couple comments to explain, we can create an Issue and document it later.

Reply:
Yes, good point! This does need updating depending on the user's queries. It's by no means complete, I just filled it with whatever was needed for my pre-written CIS-extraction wrappers.
I explain the use of this osm.config.ini file now in the tutorial, with commit 377507b6

Reply:
great!